sos/report/plugins/aide.py

```diff
@@ -0,0 +1,51 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class Aide(Plugin):
+
+    short_desc = 'Advanced Intrusion Detection Environment'
+
+    plugin_name = "aide"
+    profiles = ('system', 'security')
+
+    packages = ('aide',)
+
+    def setup(self):
+        if self.get_option("all_logs"):
+            self.add_copy_spec([
+                "/var/log/aide/",
+            ])
+        else:
+            self.add_copy_spec([
+                "/var/log/aide/aide.log"
+            ])
+
+
+class RedHatAide(Aide, RedHatPlugin):
+    def setup(self):
+        super(RedHatAide, self).setup()
+        self.add_copy_spec([
+            "/etc/aide.conf",
+            "/etc/logrotate.d/aide"
+        ])
+        self.add_cmd_output("aide -c /etc/aide.conf --config-check")
```

Comment:
This will generate an empty output and empty file if there are no inconsistencies in the database, right?

Reply:
Hey! It's only doing a linting of the config file, not anything with the database (that's `--check, -C`), but you are right that if the config file has no syntax errors then the output and resultant file will be empty.